features/networking/ovn.feature

```diff
@@ -0,0 +1,212 @@
+      | ["spec"]["hostNetwork"]           | true                                                         |
+      | ["spec"]["restartPolicy"]         | "Never"                                                      |
+    Then the step should succeed
+    And the pod named "iperf-client" becomes ready
+
+    When admin deletes the ovnkube-master "south" leader
+    Then the step should succeed
+    When I store the ovnkube-master "south" leader pod in the :new_south_leader clipboard
+    Then the step should succeed
+    And the expression should be true> cb.south_leader.name != cb.new_south_leader.name
+    And admin waits for all pods in the "openshift-ovn-kubernetes" project to become ready up to 60 seconds
+    Given I use the "<%= cb.iperf_project %>" project
+    When the pod named "iperf-client" status becomes :succeeded within 120 seconds
+    And I run the :logs client command with:
+      | resource_name | iperf-client |
+    Then the step should succeed
+    And the output is parsed as JSON
+    Then the expression should be true> @result[:parsed]['end']['sum']['lost_percent'].to_f < 10
+    Then the expression should be true> @result[:parsed]['end']['sum']['bytes'].to_f > 1024
+    Then the expression should be true> @result[:parsed]['end']['sum']['packets'].to_f > 0
+    Then the expression should be true> @result[:parsed]['end']['sum']['jitter_ms'].to_f < 1
+    And I run the :logs client command with:
+      | resource_name | iperf-server |
+    Then the step should succeed
+    And the output is parsed as JSON
+    Then the expression should be true> @result[:parsed]['end']['sum']['lost_percent'].to_f < 10
+    # server doesn't count bytes
+    Then the expression should be true> @result[:parsed]['end']['sum']['packets'].to_f > 0
+    Then the expression should be true> @result[:parsed]['end']['sum']['jitter_ms'].to_f < 1
+
+
+  # @author rbrattai@redhat.com
+  # @case_id OCP-26089
+  @admin
+  @destructive
+  Scenario: New raft leader should be elected if existing leader gets deleted or crashed in hybrid/non-hybrid clusters
+    Given admin uses the "openshift-ovn-kubernetes" project
+    When I store the ovnkube-master "north" leader pod in the clipboard
+    Then the step should succeed
+    Given admin ensures "<%= cb.north_leader.name %>" pod is deleted from the "openshift-ovn-kubernetes" project
+    Then the step should succeed
+    And I wait up to 30 seconds for the steps to pass:
+    """
+    When I store the ovnkube-master "north" leader pod in the :new_north_leader clipboard
+    Then the step should succeed
+    And the expression should be true> cb.north_leader.name != cb.new_north_leader.name
+    """
+    And admin waits for all pods in the project to become ready up to 60 seconds
+
+
+  # @author rbrattai@redhat.com
+  # @case_id OCP-26091
+  @admin
+  @destructive
+  Scenario: New raft leader should be elected if SB db on existing master is crashed
```

Comment:
@rbbratta Can we do the same for NB db too in this same case? I guess that would also be a great coverage. NB db maintains the list of all pod,service states and we can make sure that its recreating itself successfully after an accidental crash. I can modify this case to include NB db check as well, WDYT? Guess i missed it somehow while drafting this use case but checked manually its fine.

Reply:
added NB test below

Reply:
Thanks @rbbratta for taking care of that. Would appreciate if you can also update the Scenario title as per Polarion for the same.

Reply:
was wondering it the title changed, but forgot to check.

updated.